desktop-environment.el

```diff
@@ -41,6 +41,60 @@
   :group 'environment)
 
 
+;;; Customization - keyboard backlight
+
+(defcustom desktop-environment-keyboard-set-command (concat "dbus-send "
```

Comment:
I would prefer that you use the `dbus` elisp libraries instead of an external command. The code should become easier to maintain (and probably faster as well which is important because I tend to press multimedia keys quite quickly :-)).

To maintain the possibility for users to change the command, I would make `desktop-environment-keyboard-set-command` either a function or a string. By default, the value will be a function pointing towards your code calling the dbus api.

What do you think?

Reply:
I significantly changed the way argument is passed to the D-Bus. It now uses Emacs API.  

Regarding the users capacity to change the command, I have not implemented this feature. In an additional commit, I could make that happen.

Reply:
> Regarding the users capacity to change the command, I have not implemented this feature. In an additional commit, I could make that happen.

let's keep it simple on this PR so we can merge your work fast. You could still create another PR when this one is merged if you want to. That would be awesome.